src/main/java/edu/harvard/iq/dataverse/authorization/providers/oauth2/AbstractOAuth2AuthenticationProvider.java

```diff
@@ -116,6 +116,10 @@ public OAuth2UserRecord getUserRecord(String code, String state, String redirect
         final OAuthRequest request = new OAuthRequest(Verb.GET, userEndpoint, service);
         request.addHeader("Authorization", "Bearer " + accessToken.getAccessToken());
         request.setCharset("UTF-8");
+
+        // Microsoft
+        request.addHeader("Accept", "application/json");
```

Comment:
I'm slightly concerned about this because doesn't ORCID use XML instead of JSON?

Reply:
13:48
Hi!

We use this line because Microsoft identity platform needs this header when send 
an authentication request. If this header is out, we receive an error.

![image](https://user-images.githubusercontent.com/25359497/65272398-980e8d80-dae4-11e9-9b1c-c8b26660a6ce.png)

We work in a fix for the ORCID XML problem.

Reply:
@Gerafp perfect. Thanks! Please let us know when that fix is in.

Reply:
Thanks @Gerafp - I'll move this PR back to Community Dev.

Reply:
Hi.

We add a instruction for manage the authentication request when use Microsoft.

The branch has updated with our fix.

:D